data/presets/amenity/veterinary_pharmacy.json

```
@@ -0,0 +1,38 @@
+{
+    "icon": "temaki-veterinary_care",
+    "fields": [
+        "name",
+        "operator",
+        "address",
+        "building_area_yes",
+        "opening_hours"
```

See https://github.com/openstreetmap/id-tagging-schema/pull/1451#issuecomment-2680565751 for why I suggest this change.

```suggestion
        "{amenity/pharmacy}"
```

Documentation about this feature at https://github.com/ideditor/schema-builder?tab=readme-ov-file#fieldsmorefields and under the search term "reference" on that page.